### ISSUE TITLE
iris example

### ISSUE THREAD

**Issue by SamGG** [jlmelville/uwot#25]
Thanks for this great implementation.
To be fair, the species column should be removed from the example using iris, as it is the ground truth.

**Comment by jlmelville** [comment]
Thank you for the kind words.

To be clear, any non-numeric columns (including factors, like the `Species` column in the `iris` case) are automatically removed from the data before any further processing. This is a simple convenience for the typical case (in my experience) where you often want to use all the numeric columns and not have to worry about filtering out classification labels and other stuff ahead of time.

**Comment by SamGG** [comment]
Thanks a lot for this explanation. I am going to be lazy with such a feature...
Maybe you should add a specific at the end of the line of this first example for people like that are going directly to examples.
When adding the species column, all species are separated clearly.
```
set.seed(0)
iris_umap <- umap(cbind(iris[,1:4], as.numeric(iris$Species)), n_neighbors = 50, learning_rate = 0.5, init = "random")
plot(iris_umap, col = iris[,5])
```
